spec/ngp_van/client/stories_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+module NgpVan
+  class Client
+    RSpec.describe Stories do
+      let(:client) { NgpVan::Client.new }
+
+      describe '#create_stories' do
```

Comment:
@kinsomicrote Would you mind singularizing this?
```suggestion
      describe '#create_story' do
```

Reply:
@christopherstyles done